BangazonWebApp/BangazonWebApp/Models/Employees.cs

```diff
@@ -0,0 +1,41 @@
+﻿//Author: Lauren Richert
+//Purpose: To reference the Employees table and the values
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonWebApp.Models
+{
+    public class Employees
+    {
+        public int Id { get; set; }
+
+        [Required]
+        [Display(Name = "First Name")]
+        public string FirstName { get; set; }
+
+        [Required]
+        [Display(Name = "Last Name")]
+        public string LastName { get; set; }
+
+        [Required]
+        [Display(Name = "Department")]
+        public int DepartmentsId { get; set; }
+
+        [Required]
+        [Display(Name = "Assigned Computer")]
+        public int ComputersId { get; set; }
+
+        [Required]
+        [Display(Name = "Training Programs")]
+        public int TrainingProgramsId { get; set; }
```

Comment:
See above- ticket #5 specifies you should see a list of training programs, so you don't need a property called `TrainingProgramsId`

Reply:
(The list right under it is correct!)